app/quality-check/get-quality-checked-payment-requests.js

```diff
@@ -34,7 +34,7 @@ const getQualityCheckedPaymentRequests = async () => {
   return qualityCheckedPaymentRequests.map(x => {
     return {
       paymentRequest: x.paymentRequest.get({ plain: true }),
-      paymentRequests: x.paymentRequests.map(y => y.get({ plain: true }))
+      paymentRequests: x.paymentRequests
```

Comment:
This mapping is no longer needed as the fix above has already applied this step.